src/models/privacy-response.ts

```diff
@@ -1,21 +1,10 @@
 /* eslint-disable camelcase */
-import { ACTION } from './priv-terms.js';
-
-// FIXME: Move this into it's own class and refine once API is fully defined
-export interface DemandResponse {
-  demand_id: string;
-  date: string;
-  requested_action: ACTION;
-  status?: string;
-  answer?: Object;
-  message?: string;
-  lang?: string;
-  data?: Object;
-}
+import { DemandResponse } from './demand-response.js';
 
 export interface PrivacyResponse {
```

Comment:
@m4rk055 I'm wondering if it may be useful to the client app as it has the requirement of letting the user view the status of multiple requests that have made, each potentially with multiple demands